pkg/templates/template-handler.go

```diff
@@ -63,4 +66,5 @@ func addLabels(vm *kubevirtv1.VirtualMachine, template *templatev1.Template) {
 		vm.ObjectMeta.SetLabels(labels)
 	}
 	labels[templateNameLabel] = template.GetObjectMeta().GetName()
+	labels[templateNamespace] = template.GetObjectMeta().GetNamespace()
```

Comment:
This caused an issue to validate the VM on creation after vmSpec was created by tempalte-validator.
Please check if the entire process can be completed on an a cluster that runs OCP + common templates that can be resolved.

See commit #158 3fa098ed8fb4758f44029fec2579831c51a3eb7c

Reply:
OK, I will remove